subsys/net/l2/openthread/openthread.c

```diff
@@ -122,10 +122,14 @@ static void ipv6_addr_event_handler(struct net_mgmt_event_callback *cb,
 		return;
```

Comment:
Two typos in commit msg `s/withing/within'

Reply:
Thanks, fixed